extra/utils/Permissions.js

```diff
@@ -15,3 +15,5 @@ export default function hasPermission(moduleName,role,permissionType)
     }
     return true;
 }
+
+>>>>>>> 146c3daaffaa716bb8480bc3e09ba73da5e7895f
```

Comment:
Remove the unnecessary/duplicate code

Reply:
I removed all the unnecessary file and duplicate code which is present in that files.